create_session_parquet_files.py

```diff
@@ -0,0 +1,61 @@
+import os
+import argparse
+import re
+
+from astropy.table import Table, vstack
+import pandas as pd
+import numpy as np
+
+# regular expression to match the session name given a path to antenna FITS file
+SESSION_REGEX = re.compile(r"/[^/]*/Antenna")
+
+def group_files_df(manifest_path: str):
+    """Returns a pd dataframe with one column containing all the paths listed in the manifest file, and
+    another column containing each file's associated session"""
+    lines = open(manifest_path, "r").readlines()
```

Comment:
`readlines()` forces a full iteration through all rows in the file before any manipulations can be done; better to do all in a single pass as per below